permissions/src/iosMain/kotlin/dev/icerock/moko/permissions/ios/PermissionsController.kt

```diff
@@ -41,7 +41,7 @@ class PermissionsController : PermissionsControllerProtocol {
             Permission.CAMERA -> AVCapturePermissionDelegate(AVMediaTypeVideo, permission)
             Permission.GALLERY -> GalleryPermissionDelegate()
             Permission.STORAGE, Permission.WRITE_STORAGE -> AlwaysGrantedPermissionDelegate()
-            Permission.LOCATION, Permission.COARSE_LOCATION ->
+            Permission.LOCATION, Permission.COARSE_LOCATION, Permission.BACKGROUND_LOCATION ->
```

Comment:
can you attach link that background mode location permissions on iOS works with same permission like other?

Reply:
> Note
> 
> The system allows background updates for apps with either When in Use or Always authorization. If an app isn’t running when an update occurs, the system launches the app only if it has Always authorization and uses the significant location change, visits, or region monitoring services.

from https://developer.apple.com/documentation/corelocation/handling_location_updates_in_the_background